### PR TITLE
Add missing requirement

### PR DESCRIPTION
**Pull request by rafalgajda-silvair**
This patch aligns the VEN behavior with specification and adds method for opting out of the event.

### CODE REVIEW

**Review by bartoszflis-silvair — COMMENTED** [review]
Do we have any unit tests for that?

File: oadr2/event.py [hunk 1]
```diff
@@ -447,6 +459,22 @@ def remove_events(self, evt_id_list):
...
+
+    def optout_event(self, e_id):
+        '''
+        Opt out of an event by its ID
+
+        :param e_id: ID of the event we want to opt out of
+        :return:
+        '''
+
+        if e_id not in self.db.get_active_events().keys():
```
Comment (michallowasrzechonek-silvair) [inline oadr2/event.py]: I don't think `.keys()` is needed here:

```
>>> 1 in {1: "one", 2: "two"}
True
>>> 3 in {1: "one", 2: "two"}
False
```

File: oadr2/event.py [hunk 2]
```diff
@@ -447,6 +459,22 @@ def remove_events(self, evt_id_list):
         event_id_list - List of Event IDs 
         '''
         self.db.remove_events(evt_id_list)
+        for evt in evt_id_list:
+            if evt in self.optouts:
```
Comment (michallowasrzechonek-silvair) [inline oadr2/event.py]: https://docs.python.org/3.7/library/stdtypes.html#frozenset.discard

**Review by michallowasrzechonek-silvair — CHANGES_REQUESTED** [review]

File: oadr2/event.py (hunk 2 above)
Comment (rafalgajda-silvair) [reply oadr2/event.py]: Changed

File: oadr2/event.py (hunk 1 above)
Comment (rafalgajda-silvair) [reply oadr2/event.py]: Changed

**Review by michallowasrzechonek-silvair — APPROVED** [review]

**Pull request merged**